waiter/src/waiter/core.clj

```diff
@@ -1754,7 +1772,7 @@
                            (token/handle-token-request
                              clock synchronize-fn kv-store token-cluster-calculator token-root history-length limit-per-owner
                              waiter-hostnames entitlement-manager make-inter-router-requests-sync-fn validate-service-description-fn
-                             attach-service-defaults-fn request))))
+                             attach-service-defaults-fn (:tokens-update-chan token-watch-maintainer) request))))
```

Comment:
Nitpick, extract this in a let-block, `{:keys [tokens-update-chan]} token-watch-maintainer`.